.github/actions/create-or-update-comment/action.yml

```diff
@@ -0,0 +1,36 @@
+name: openverse/create-or-update-comment
+description: Thin wrapper around peter-evans/create-or-update-comment that skips commenting on forks and instead prints the message body to the console.
+
+inputs:
+  # See original action documentation for descriptions of these
+  # inputs: https://github.com/peter-evans/create-or-update-comment#action-inputs
+  comment-id:
+    default: null
+  edit-mode:
+    default: null
+  body:
+    required: true
+  issue-number:
+    default: null
+
+runs:
+  using: "composite"
+  steps:
+    - name: Call upstream action
+      # Not fork
+      if: github.repository == 'WordPress/openverse'
+      uses: peter-evans/create-or-update-comment@v2
```

Comment:
Something I'm not sure of - does/would dependabot update versions within composite actions? I'm trying to find an answer now.

Reply:
Update, I found this link: https://stackoverflow.com/a/74997644/3277713

TL;DR is that we have to add explicit dependabot checks for all our composite action directories 😞

Reply:
Good call, easy enough to add.